src/plugins/dashboard/public/url_generator.ts

```diff
@@ -73,6 +73,11 @@ export type DashboardAppLinkGeneratorState = UrlGeneratorState<{
    * true is default
    */
   preserveSavedFilters?: boolean;
+
+  /**
+   * View mode of the dashboard.
+   */
+  viewMode?: 'edit' | 'view';
```

Comment:
There is a `ViewMode` type somewhere, could we reuse it for sanity?

Reply:
Updated in https://github.com/elastic/kibana/pull/68784/commits/457172c740350ce3e9aa65c23b445ddf2dcf7287